src/controllers/KeyCDNController.ts

```diff
@@ -41,6 +41,36 @@ export default class KeyCDNController implements ICacheController {
       });
   }
 
+  public get flushURLsConfig(): Hapi.IRouteAdditionalConfigurationOptions {
+    return {
+      handler: this.flushURLs.bind(this),
+      tags: ['api', 'cache', 'keycdn'],
+      description: 'Flush KeyCDN cache URL',
+      validate: {
+        params: this.paramsSchema,
+        payload: this.payloadSchema.keys({
+          urls: Joi.array().items(Joi.string()).min(1).required()
```

Comment:
is this secure if array is empty ?

Reply:
Here it says the `urls` array must have at least 1 elements. An empty array is not accepted.